.github/workflows/jobs-compile-and-test.yml

```diff
@@ -0,0 +1,51 @@
+# Reuable workflow for compiling and testing extension.
+name: Compile and test extension
+
+on:
+  workflow_call:
+    inputs:
+      runner-env:
+        required: true
+        type: string
+      yarn-args:
+        type: string
+
+
+jobs:
+  build:
+    runs-on: ${{ inputs.runner-env }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Use Node.js 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Install Dependencies
+        run: yarn install ${{ inputs.yarn-args }}
+        working-directory: Extension
+
+      - name: Compile Sources
+        run: yarn run compile

```

Comment:
@fearthecowboy Are these still the correct command lines to use when building in the lab?